kvui.py

```diff
@@ -386,6 +386,60 @@ def insert_text(self, substring, from_undo=False):
         return super(ConnectBarTextInput, self).insert_text(s, from_undo=from_undo)
 
 
+def is_command_input(string: str) -> bool:
+    return string and string[0] in "/!"
+
+
+class CommandPromptTextInput(TextInput):
+    MAXIMUM_HISTORY_MESSAGES = 50
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
```

Comment:
When I go look at the base class implementation, it looks like this:
```python
    def __init__(self, **kwargs):
```
maybe we should match and not have `*args` here?